server/sync_watcher.go

```diff
@@ -272,13 +272,8 @@ func (watcher *SyncWatcher) processSyncWatch(ctx context.Context, path string) e
 				if response.Err != nil {
 					return response.Err
 				}
+				watcher.watchExtensionHandler(response)
 
-				watcher.queue.Add(job.NewJob(
```

Comment:
I think it is really rare to control this kind of finely granular logic therefore I do not optionize it. What do you think? @yudai